docs/bots.md

```diff
@@ -54,3 +54,26 @@ Example of creating the bot user with the API.
 ```sh
 curl -X POST -H "content-type: application/json" -H "x-rems-api-key: 42" -H "x-rems-user-id: owner" http://localhost:3000/api/users/create --data '{"userid": "rejecter-bot", "name": "Rejecter Bot", "email": null}'
 ```
+
+## Bona Fide bot
+
+User id: `bonafide-bot`
+
+The Bona Fide bot is used for granting peer-verified GA4GH
+ResearcherStatus visas. See [ga4gh-visas.md](ga4gh-visas.md) for more
+background.
+
+The Bona Fide bot is designed to be used with
+- a _default workflow_ that has the bot as a handler (and optionally some human handlers)
+- a catalogue item that has a form that has an email field
```

Comment:
Maybe we should document here that there must be only one email field (or if we change so that the _first_ will always be used)?

Reply:
I'll document the current behaviour for now.